_releases/2024-07-09-1.13.0-released.md

```diff
@@ -17,6 +17,12 @@ This release includes [XX changes since 1.12.2](https://github.com/crystal-lang/
 by ZZZ contributors. We thank all the contributors for all the effort put into
 improving the language! ❤️
 
+## Advances in multi-threading support
+
+Since we started to [improve multi-threading support](/2024/02/09/84codes-manas-mt) with the help of [84codes], each new release gets several improvements and refactors related to it. Within the list of changes we list the most prominent from a user point of view, but several others can be found in the [changelog](https://github.com/crystal-lang/crystal/releases/tag/1.13.0).
+
+More importantly, the current implementation of the [corresponding RFC](https://github.com/crystal-lang/rfcs/pull/2) can be tested by eager developers by following simple from the [`execution_context`](https://github.com/ysbaddaden/execution_context)  shard. This library is expected to be part of the compiler once we are certain it's ready.  We're still ironing it, so bugs are expected, and reports very much welcomed!
```

Comment:
**Suggestion:** The wording sounds very generic. Maybe we could tailor it more towards this specific release?

```suggestion
As part of the [project to improve multi-threading support](/2024/02/09/84codes-manas-mt) with the help of [84codes], 
this release includes several improvements and refactors related to concurrency and the event loop. So far, these have been mostly internal changes with minor relevance from a user's point of view. We expect more significant changes in the next release.

A preview of the upcoming execution context ([RFC](https://github.com/crystal-lang/rfcs/pull/2)) is available as a standalone shard: [`ysbaddaden/execution_context`](https://github.com/ysbaddaden/execution_context). It's available for testing by eager developers. This library is expected to be part of the compiler once we are certain it's ready.  We're still ironing it, so bugs are expected, and reports very much welcomed!
```